src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -26,36 +29,44 @@ public class ViewLoadedParams
         // Here's a start on the implementation
         //
 
```

Comment:
Let's remove all of the sub comment up to the line that says `TBD`.  These comments are useless after this point.